alws/build_planner.py

```diff
@@ -260,6 +260,10 @@ async def _add_single_ref(
             ref_platform_version: typing.Optional[str] = None):
         for platform in self._platforms:
             arch_tasks = []
+            arch_list = self._request_platforms[platform.name]
+            if 'i686' in arch_list and arch_list.index('i686') != 0:
+                arch_list.insert(0, arch_list.pop(arch_list.index('i686')))
```

Comment:
If I were you and you asked me the line one month late I wouldn't tell me what is going exactly on here :) Suppose, you want to set on the 1st place i686 arch if it exists, but... But we could rewrite it more clearly

Reply:
You right, i want to set on the 1st place i686 arch if it exists, but right now i don't know how to make it more clearly)
I can add comment `TODO: rewrite more clearly`